discord/handler.py

```diff
@@ -148,6 +148,15 @@ def get_token_for_athlete(id):
 
 
 def post_message(body):
+    # See https://developers.strava.com/docs/reference/#api-models-ActivityType
+    activity_colours = {
+        'Run': int('fc4c02', 16),  # orange
```

Comment:
I'd prefer this format: `'Run': 0xfc4c02`

Reply:
Okay. It would require another conversion step though.

Reply:
`0x fc4c02` will return a base10 in representation of the hex value, so you should be able to use it without any other conversion.

Reply:
Ah. Awesome.